### PR TITLE
Improve wide review handling

### PR DESCRIPTION
**Pull request by plehegar**
This clarifies further how to get wide review (#12).

* clarify that publishing an updated technical report is expected as part of getting wide review, to discourage Groups to use their editor's drafts for that purpose (and it amplifies the reach to the public).
* our tooling will pick up if a publication is interested in wide review.
* as a tip, Groups should consider creating a GitHub issue to track the state of an ongoing wide review.

Hopefully this helps.

cc @nigelmegitt @frivoal @fantasai

-----

[Preview](https://htmlpreview.github.io/?https://raw.githubusercontent.com/w3c/documentreview/wide-review-42/index.html), [HTML diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FGuide%2Fdocumentreview%2F&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fw3c%2Fdocumentreview%2Fwide-review-42%2Findex.html)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/documentreview/pull/21.html" title="Last updated on Oct 26, 2023, 5:47 PM UTC (851fb38)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/documentreview/21/3393806...851fb38.html" title="Last updated on Oct 26, 2023, 5:47 PM UTC (851fb38)">Diff</a>

### CODE REVIEW

**Comment by plehegar** [comment]
(with https://github.com/w3c/documentreview/pull/15 merged, the tip could even be more easily fullfilled)

File: index.html
Comment (r12a) [inline index.html]:
```suggestion
  <p>Wide review should or must be requested:</p>
```

File: index.html
Comment (r12a) [inline index.html]:
```suggestion
  <li>After a <a href="https://www.w3.org/Consortium/Process/#first-wd">First
```

File: index.html
Comment (r12a) [inline index.html]:
```suggestion
  Public Working Draft</a> is published (for most documents).
```

File: index.html
Comment (r12a) [inline index.html]:
```suggestion
  allow changes where issues are identified.</li>
```

File: index.html
Comment (r12a) [inline index.html]:
```suggestion
  review (perhaps requesting a review limited to the new features).</li>
```

File: index.html
Comment (r12a) [inline index.html:58]: Though there are some useful additions, a couple of things we put in on purpose were dropped.  I'd also like to reorganise the text slightly, so that the list front-ends the target audiences better and makes it easier to see as a list.  I'll make a suggestion shortly.

Comment (plehegar) [reply index.html:58]: feel free to make the changes in my branch

Comment (r12a) [reply index.html:58]: @plehegar I'm in a bit of a rush, so i'll just paste the source code for my proposed revision of the "Who to ask..." section below.  This is for the whole section. hth

```
<section id="who_to_ask_for_review">
<h2>Who to ask for wide review?</h2>

<p>The objective is to ensure that the entire set of stakeholders of the Web community, including the general public, have  adequate notice of the progress of the Working Group and are able to actually perform reviews of and provide comments on the specification. When considering <a href="https://www.w3.org/Guide/transitions">requests to advance the maturity level of the document</a>, the Director will consider who has been explicitly offered a reasonable opportunity to review the document.</p>

<p>You must publish an updated technical report, with the Status of the Document indicating clearly that you're looking for <em>wide review</em>, before inviting review from other stakeholders.<strong> </strong>Our <a href='https://github.com/w3c/transition-notifier/blob/main/notify.js'>tooling</a> picks up on publications and propagates notifications to <a href="mailto:public-review-announce@w3.org">public-review-announce@w3.org</a> automatically (for example, <a href='https://lists.w3.org/Archives/Public/public-review-announce/2021Jun/0008.html'>Candidate Recommendation Snapshot: Payment Request API (Call for Wide Review)</a>).</p>
<p>You should then reach out to:</p>
<ul>
<li>the groups listed in the WG's charter, especially those who manage dependencies.</li>
<li>the groups jointly responsible for a particular document (if any).</li>
<li>the <a href="#how_to_get_horizontal_review">horizontal groups</a>.  If it appears that one of those is not relevant (even if not listed in your charter), talk to them informally to confirm that. <em>Note that sending an email to the public-review-announce list alone is not sufficient to trigger horizontal reviews. You will still need to formally request review from the Horizontal Groups, as described below</em></li>
<li>the general public,  including Web developers, technology providers and implementers, application developers, etc. Consider:
    <ul>
      <li>sending a dedicated announcement to <a href="mailto:public-review-announce@w3.org">public-review-announce@w3.org</a> if needed (in case the default notice sent to that list is not enough).</li>
      <li>using blog posts, social media, or other ways of alerting the public to your document and requesting feedback.</li>
      </ul>
      </li>
<li>other groups, at your discretion, even if not in the WG charter, such as other W3C groups, external organizations and SSOs  working on related areas, etc.</li>
</ul>

<p><strong>Tip:</strong> consider tracking your wide review progress using a GitHub issue, such as <a href="https://github.com/w3c/sensors/issues/299">issue #299 of the Sensors API</a>. You can then simply point the Director to the issue.</p>


<p>The reviews provided by the <a href="https://www.w3.org/Guide/process/charter.html#horizontal-review">horizontal
groups</a>, a subset of a full wide review, do receive special attention and much of the rest of this document focuses on how and when to conduct horizontal reviews.</p>
</section>

```

File: index.html
Comment (nigelmegitt) [inline index.html]: "gotten" is a colloquialism. 

```suggestion
  <li>When new features are added after a document has already received wide
```

File: index.html
Comment (nigelmegitt) [inline index.html]:
```suggestion
progress of the Working Group and were genuinely able to perform reviews of and
```

File: index.html
Comment (nigelmegitt) [inline index.html]:
```suggestion
    monitors publications and propagates to <a
```

File: index.html
Comment (nigelmegitt) [inline index.html]:
```suggestion
  <li>Inform, and request review from, the Groups listed in the WG's charter, especially those who
```

File: index.html
Comment (nigelmegitt) [inline index.html]:
```suggestion
  <li>Inform, and request review from, to the Groups jointly responsible for a particular document (if any).</li>
```

File: index.html
Comment (nigelmegitt) [inline index.html]:
```suggestion
  <li><a href="#how_to_get_horizontal_review">Request reviews</a> from the
      <a href="https://www.w3.org/Guide/process/charter.html#horizontal-review">horizontal groups</a></li>
```

File: index.html
Comment (nigelmegitt) [inline index.html]:
```suggestion
  <li>Actively inform the general public. Beyond the publication, you do so by:
```

File: index.html
Comment (nigelmegitt) [inline index.html]:
```suggestion
<p>In addition, you should consider contacting other groups, at your
```

**Review by nigelmegitt — CHANGES_REQUESTED** [review]
Looks like a good improvement. Made a bunch of mainly language-based suggestions to get away from using US-centric colloquialisms like "gotten" and "reach out" in the hope of reaching more geographically neutral English language, and being clearer about the intent.

**Review by nigelmegitt — COMMENTED** [review]
All my changes incorporated, thanks @plehegar - it looks like @r12a 's comments have also been incorporated, but I'm not clear because the comment is not shown as resolved. Changing my review to a "Comment" so it doesn't block merging, happy to mark as Approved after seeing it when it seems to be stable with respect to other reviewers' comments.

File: index.html
Comment (plehegar) [reply index.html:58]: Integrated in https://github.com/w3c/documentreview/pull/21/commits/c00eb79895d88ad4d4c9f90d3ee4a7e97e17c362

**Comment by plehegar** [comment]
Alright, I believe I folded everything in. Next step are to rebase with the main branch, tweak to integrate the recent github template changes, get a +1 from @swickr , give it a few days in case other comments are coming, merge, and finally announce the update to the Chairs list.
big kuddos to @r12a @nigelmegitt for the quick turn-around!

File: index.html
Comment (nigelmegitt) [inline index.html]: This has reverted the fix you made based on my earlier comment.

```suggestion
 of the Working Group and are genuinely able to perform reviews of and provide
```

File: index.html
Comment (nigelmegitt) [inline index.html]:
```suggestion
<p>You should then inform, and request reviews from:</p>
```

File: index.html
Comment (nigelmegitt) [inline index.html]:
```suggestion
<li>the <a href="https://www.w3.org/Guide/process/charter.html#horizontal-review">horizontal groups</a>
using the <a href="#how_to_get_horizontal_review">method described below</a>.  If it
```
<a href="https://www.w3.org/Guide/process/charter.html#horizontal-review">horizontal groups</a>

File: index.html
Comment (nigelmegitt) [inline index.html]: Not sure informal contact is adequate here for demonstrating HR as per the process? Isn't a "paper" trail needed that shows that the WG told the HR group about the change and the HR group said they had no comments?

**Review by nigelmegitt — CHANGES_REQUESTED** [review]
2nd iteration!

File: index.html
Comment (plehegar) [reply index.html]: By default, the paper trail is expected but the Director did use judgment in the past. For example, we don't really expect an Accessibility review on the WebAssembly Core specification. @swickr , wdyt?

Comment (r12a) [reply index.html]: If i18n feels that a review is needed, they will ask the WG to start the formal process.  If they agree that it's not needed, then we'd like to avoid unnecessary paperwork or process steps (mainly because we want to preserve as much bandwidth as possible).

File: index.html
Comment (r12a) [inline index.html]:
```suggestion
<script src="createGitHubIssue.js"></script>
```

To avoid warnings in HTML validator.

File: index.html
Comment (nigelmegitt) [reply index.html]: The Process does not seem to allow for WG's _not_ requesting review. I understand from the HR group's perspective that they don't want unnecessary work; on the other hand, from a WG's perspective, they want to have a formal statement of "we did what we were required to do". 

I wonder: is it actually less work to have the informal conversation than to review the formal request and respond with a "go ahead, no input from us"?

Comment (r12a) [reply index.html]: Nigel, i think this process related suggestion doesn't belong in this PR. Could you raise an issue?

Comment (nigelmegitt) [reply index.html]: Richard, I don't think I'm suggesting any change to the Process, I'm just trying to get the wording in this PR to match what folk need to do to meet the process as it stands. Maybe we need to wait for @swickr to respond to [@plehegar 's clarification question](https://github.com/w3c/documentreview/pull/21#discussion_r662476772) before we can proceed.

Comment (plehegar) [reply index.html]: I concur here that we should set proper expectation on the amount of paper trail needed. It's easier for the Director to conclude that no review was needed by i18n when there is an actual record from i18n, than relying on out of band conversation and uncheckable assertions. So, let's wait for @swickr input.

Comment (swickr) [reply index.html]: > It's easier for the Director to conclude that no review was needed by i18n when there is an actual record from i18n, than relying on out of band conversation and uncheckable assertions.

Yep; explicitly recorded assertions are much preferred; thanks.

**Comment by plehegar** [comment]
[HTML diff with main](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FGuide%2Fdocumentreview%2F&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fw3c%2Fdocumentreview%2Fwide-review-42%2Findex.html)

File: index.html
Comment (plehegar) [reply index.html]: Addressed in https://github.com/w3c/documentreview/pull/21/commits/89e323b984966f3cee7c8b5de5d128b48fcc4fef

**Review by swickr — APPROVED** [review]
Great work on the text.  Big improvement.  Thanks @plehegar, @nigelmegitt , @r12a.

File: index.html
Comment (r12a) [reply index.html]: Just a nit: i wrote 'process' with a lowercase 'p', ie. referring to the general process, rather than the Process document.  Probably doesn't change anything.

**Comment by samuelweiler** [comment]
This is starting to get long enough that it gets some things muddled.  For example, there's this new section on when to ask for **wide** review as well as the existing verbiage in the HR section re: when to ask for **HR** review.

If we're breaking out the "when", perhaps we need to pull it out of the HR section, also?  And, in case, this is getting long enough that we need some combination of TL;DR version or at least an early-in-the-doc clarification of what we mean by wide v. horizontal review.  Or we need to see if we can clarify with a net _reduction_ in words.

**Review by samuelweiler — CHANGES_REQUESTED** [review]
This is starting to get long enough that it gets some things muddled.  For example, there's this new section on when to ask for **wide** review as well as the existing verbiage in the HR section re: when to ask for **HR** review.

If we're breaking out the "when", perhaps we need to pull it out of the HR section, also?  And, in case, this is getting long enough that we need some combination of TL;DR version or at least an early-in-the-doc clarification of what we mean by wide v. horizontal review.  Or we need to see if we can clarify with a net _reduction_ in words.

**Comment by r12a** [comment]
> If we're breaking out the "when", perhaps we need to pull it out of the HR section, also? 

I'd prefer to leave it, as it's worth saying twice, and more likely to be noticed by those who just jump to the 'How to get horizontal review' section.  But actually, i think it contains some additional information that is HR specific.

> Or we need to see if we can clarify with a net reduction in words.

I suggest zapping the "Terminology & abbreviations" and "Enhancement Requests" sections.  The former is a hangover from the original document that we never looked at closely.  The second is about a link that doesn't actually work anyway.

I'd also be happy to drop the FAQ section.  That, too, is very old text that we never examined during the rewrite.

That would produce a good net reduction of words on the page.

**Comment by plehegar** [comment]
(waiting on a review from @frivoal )

**Comment by plehegar** [comment]
> Some updates deserve announcement, and others do not. It is good to
> differentiate this, and make announcements appropriately. But it's not
> appropriate to put that information in the SOTD, because how people
> should treat the document published 2 days later shouldn't differ when
> it's essentially the same with some minor fixes.

People don't know that the nature of changes. They see 2 documents with 2 different dates. Whatever dated version they picked needs to be clear to the Director. The Director needs to precisely review changes done to a specification since that spec was sent to wide review. Here is yet another messy instance in
 https://github.com/w3c/transitions/issues/373#issuecomment-916207646

> I would advise to not make this part of the SOTD, and use some other
> flag or tracker or whatever if you want tooling around it. (Fwiw, CSSWG
> has been doing these things manually for years.)

The SOTD marker is meant to help us diffuse the information and draw extract attention. I guess, once I finally get around to finish the tooling on wide review, the marker would not be needed. I'm ok with removing/relaxing the requirement in this pull request.

**Comment by fantasai** [comment]
Let me paste the full contents of my comment to which @plehegar was replying:

> * Our documents are always open for wide review, and we shouldn't be marking certain publications as requesting wide review. In particular, if Miriam pushes an update to css-cascade-5 and because it has significant changes wants to trigger a wide review, me updating the draft 2 days later to fix some minor errors shouldn't mean that it is no longer under wide review, or that the 2-days-earlier version should be the basis of wide review.
> 
> * Indeed, we shouldn't be pushing out public signals for review on every update to /TR, that would create a very low signal-to-noise ratio. Some updates deserve announcement, and others do not. It is good to differentiate this, and make announcements appropriately.
>
> But it's not appropriate to put that information in the SOTD, because how people should treat the document published 2 days later shouldn't differ when it's essentially the same with some additional fixes.

And now my reply to https://github.com/w3c/documentreview/pull/21#issuecomment-916230129 is as follows:

When we send a request for wide review -- to the public or to an HRG -- we want to be linking to the Latest Version URL so that the reviewers, on the day they are reviewing, see our latest work. We do not want to be linking to a dated copy, because we don't want the reviewers to be working off of stale documents.

If the Director wants to know what version was reviewed by an HRG, then that information should be requested from the HRG in their response to the review: please link to the dated URL of the version that you reviewed. Alternately the Director could use the date of the HRG response to identify the dated version that was reviewed. As long as Horizontal Review is requested against /TR (as it should be), there should be no problem identifying the version of the document that was reviewed.

File: index.html
Comment (fantasai) [reply index.html]: The Process does not require sign-off from every HRG. It requires it to transition without the Director's permission, but the Director can use their judgement to approve a transition that skips certain HRG reviews (such as the unnecessary a11y review of WebAssembly mentioned earlier).

Comment (fantasai) [reply index.html]: When filing the transition request without the full set of HRG reviews, groups should probably explain why they think a particular HRG's review is unnecessary, and the Director can accept when it seems obvious and push back if he feels it's inappropriate or borderline. I think that would be a reasonable substitute to asking the HRGs to do more paperwork on things that are clearly irrelevant to their interests.

File: index.html
Comment (fantasai) [inline index.html:128]: Split the sentence after "and" here. :)

File: index.html
Comment (fantasai) [inline index.html:70]: All of our publications are inviting review, that is why we publish them. Let's not add more junk to the SOTD.

File: index.html
Comment (michael-n-cooper) [inline index.html:44]: I suggest a more precise definition of "before", such as "maximum 6 months before entering CR" or something. The time has to be reasonable to receive and process comments, but not so long that wide reviews might have gone stale yet could be considered by the definition as acceptable.

Comment (fantasai) [reply index.html:44]: A little more guidance would be helpful, but a specific time period is maybe not a good idea. The concern is the amount of churn: if the document has only had editorial improvements and minor fixes in those six months, there should be no problem with the horizontal reviews. (This happens reasonably often in the CSSWG, because there are so many specs and few editors, things swap in and out of attention; and the final stages of polishing before CR are often subtle algorithm tweaks that wouldn't impact HR.)

**Comment by samuelweiler** [comment]
We should also update the HR request templates to encourage working drafts

File: index.html
Comment (plehegar) [reply index.html:70]: Except that the other groups need a clearer signal that wide review is expected before moving to CR. It's not junk, it gets picked up by our tools for the past few years.

**Comment by plehegar** [comment]
Merging this pull request since it encodes current practices. Additional issues are welcome separately.

**Pull request merged**
